scripts/local-clusters/configs/partial/sc-node-local-dns.yaml

```
@@ -0,0 +1,20 @@
+nodeLocalDns:
+  customConfig: |-
+    $domain:53 {
+      errors
+      bind 169.254.20.10 10.96.0.10
+      template IN A $domain {
+        match "(^grafana|^harbor|^opensearch|\.ops)\.$domain\.$"
```

Was testing this with the test harness and apparently we've missed adding resolve for Dex to the SC. So this needs to be added in all DNS configs.
```suggestion
        match "(^dex|^grafana|^harbor|^opensearch|\.ops)\.$domain\.$"
```

---

fixed